bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml

```diff
@@ -66,9 +66,15 @@ metadata:
         }
       ]
     capabilities: Basic Install
+    features.operators.openshift.io/disconnected: "false"
+    features.operators.openshift.io/fips-compliant: "true"
+    features.operators.openshift.io/proxy-aware: "true"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "true"
```

Comment:
Only `proxy-aware` is set as a true infrastructure features in the previous version.  Can you explain why you mark `token-auth-aws` and `fips-compliant` as true here?

Reply:
The commit message now explains why these features are marked as true.

Reply:
What a phrase - "human obliviousness" !